### PR TITLE
Add Crewed supersonic science to cockpits

### PR DESCRIPTION
**Pull request by leudaimon**
Still untested! I'll test them properly later. As it's my first venture into Kerbalism configs, decided to put it here already in case something obvious is wrong. 
This depends on the removal of all other science from the Earth Flying low in RP-0, soon to be commited

### CODE REVIEW

**Comment by leudaimon** [comment]
I have tested it ingame, and the experiments work. I'd like some input on the values and parameters for balancing though.

**Comment by siimav** [comment]
Looking at this PR through the GH web interface, there appears to quite a few places with mixed spaces and tabs.

**Comment by leudaimon** [comment]
Oof, I was hoping my editor would just keep whatever was in the file, given I was only copy-pasting the indentations. I can try to clean it up later

File: GameData/KerbalismConfig/System/Science/Configure.cfg [hunk 1]
```diff
@@ -472,6 +472,84 @@
...
+			  id_field = experiment_id
+			  id_value = RP0-SupersonicHigh1
+			}
+		}
+		SETUP
+		{
+			name = Hypersonic flight
+			desc = //fixme
+			mass = 0.004
+			tech = supersonicFlightRP0
```
Comment (lpgagnon) [inline GameData/KerbalismConfig/System/Science/Configure.cfg:541]: Wouldn't a Hypersonic Flight experiment make more sense to unlock in the Hypersonic Flight r&d node?

Comment (leudaimon) [reply GameData/KerbalismConfig/System/Science/Configure.cfg:541]: High altitude is already there, no? If it's not, it was a mistake on my part. About Hypersonic Flight node, I could do that yeah... my idea for that is X-15-like flights.

Comment (siimav) [reply GameData/KerbalismConfig/System/Science/Configure.cfg:541]: If we do move it to hypersonic flight then 20 sci isn't worth much at all by then.

Comment (lpgagnon) [reply GameData/KerbalismConfig/System/Science/Configure.cfg:541]: high altitude: sorry yes, my turn to confuse the first two supersonic nodes

**Pull request merged**